tests/tt_metal/tt_metal/test_create_kernel_with_kernel_path_env_var.cpp

```diff
@@ -1,5 +1,6 @@
 #include <gtest/gtest.h>
```

Comment:
does this test run normally or under CI?  thinking we should update the CI script to create the dir under /tmp/metal_test and pass the env var in so it gets run.  I think this should be ok...

Reply:
@tt-rkim Can I create a CI test for this in the `All Post-commit tests` workflow or should I add it to another workflow?

Reply:
Added to `All Post-commit tests` workflow: https://github.com/tenstorrent/tt-metal/actions/runs/11003253105/job/30552388812#step:6:23